src/app/client/page.tsx

```diff
@@ -42,38 +17,22 @@ async function getAssigneeList(token: string): Promise<IAssignee> {
   return data.clients
 }
 
-async function getTokenPayload(token: string): Promise<Token> {
-  const copilotClient = new CopilotAPI(token)
-  const payload = TokenSchema.parse(await copilotClient.getTokenPayload())
-  return payload as Token
-}
-
 export default async function ClientPage({ searchParams }: { searchParams: { token: string } }) {
   const token = searchParams.token
 
-  const [workflowStates, tasks, assignee, tokenPayload] = await Promise.all([
-    await getAllWorkflowStates(token),
-    await getAllTasks(token),
-    addTypeToAssignee(await getAssigneeList(token)),
-    getTokenPayload(token),
-  ])
+  const assignee = addTypeToAssignee(await getAssigneeList(token))
+
   return (
     <>
-      <ClientSideStateUpdate
-        workflowStates={workflowStates}
-        tasks={tasks}
-        token={token}
-        assignee={assignee}
-        tokenPayload={tokenPayload}
-      >
-        <RealTime>
+      <ClientSideStateUpdate assignee={assignee}>
+        <ForceDisableServerRender>
```

Comment:
this isn;t a great solution, it removes the benefit of server rendering and the nice experience we get from loading from the server